GH-371 Added initial public files doc (#372)
adonisjs/docs: 06-Digging-Deeper/05-Public-Files.adoc

---
title: Public Files
permalink: public-files
category: digging-deeper
---

= Public Files

toc::[]

AdonisJS can serve static assets and public files

> NOTE: It would be better to use CDN or reverse proxy like Nginx / Apache etc to serve static files

Reviewer: ```suggestion
NOTE: It would be better to use a CDN or a reverse proxy(i.e Nginx or Apache) to serve static files.
```

== Setup

> To serve public files `'Adonis/Middleware/Static',` server middleware must be enabled in `start/kernel.js`

Reviewer: ```suggestion
To serve public files, the `Adonis/Middleware/Static` server middleware must be enabled in the file `start/kernel.js`.
```
